src/interface/storage.thrift

```diff
@@ -428,6 +428,22 @@ struct DropCPRequest {
     2: string                       name,
 }
 
+struct RebuildTagIndexRequest {
+    1: common.GraphSpaceID          space_id,
+    2: list<common.PartitionID>     parts,
+    3: common.TagID                 tag_id,
+    4: common.SchemaVer             tag_version,
+    5: common.IndexID               index_id,
+}
+
+struct RebuildEdgeIndexRequest {
+    1: common.GraphSpaceID          space_id,
+    2: list<common.PartitionID>     parts,
+    3: common.EdgeType              edge_type,
+    4: common.SchemaVer             edge_version,
+    5: common.IndexID               index_id,
+}
+
```

Comment:
Could we unify  RebuildTagIndexRequest and RebuildEdgeIndexRequest into RebuildIndexRequest?

Reply:
It's  a good idea :)